cli/ceph/mgr.py

```diff
@@ -10,21 +10,25 @@ def __init__(self, nodes, base_cmd):
 
         self.base_cmd = f"{base_cmd} mgr"
 
-    def module(self, action, module=None, force=False):
+    def module(self, action, module=None, force=False, **kw):
         """Disable MGR module.
 
         Args:
             action (str): module action (disable|enable)
             module (str): ceph module to be disabled
             force (bool): use `--force`
+            kw: Key/value pairs of configuration information to be used in the test.
         """
         cmd = f"{self.base_cmd} module {action}"
         if module:
             cmd += f" {module}"
         if force:
             cmd += " --force"
-
-        return self.execute(sudo=True, long_running=True, cmd=cmd)
+        if "long_running" in kw:
```

Comment:
@MohitBis this if check is not required. As we are using key=value in kw this can be passed directly to execute as -
```
return self.execute(sudo=True, cmd=cmd, kw=**kw)
```

Reply:
updated